src/pages/personalisation/EverydayThingsSubcategories.js

```diff
@@ -12,16 +12,16 @@ export default class EverydayThingsSubcategories extends BaseMultiQuestion {
         answers: {
             /* eslint-disable max-len */
             "Food packages/parcels/vouchers": append("food parcel voucher pack"),
-            "Transport": append("transport"),
+            "Transport": append("transport voucher taxi subsidy"),
```

Comment:
Will this return anything matching 'subsidy'? Do we want `'"transport voucher" "taxi subsidy"'`?